styles.css

```diff
@@ -0,0 +1,209 @@
+/* ---- GENERAL ---- */
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: 'Open Sans', sans-serif;
+}
+
```

Comment:
Consider refactoring some css classes in order to be able to apply styling to multiple html elements with the same class.